File: examples/js/get__conversation_{conversation_id}.js
const conversationId = 'YOUR_CONVERSATION_ID';
const authToken = process.env.AIOPS_TOKEN || 'YOUR_TOKEN';
// 示例：GET /conversation/{conversation_id}
fetch(`http://localhost:8000/api/v1/conversation/${conversationId}`, {
    method: 'GET',
    headers: { Authorization: `Bearer ${authToken}` }
})
.then(res => res.json())
.then(data => console.log(data))
.catch(err => console.error('Request error:', err));

File: examples/python/get__conversation_{conversation_id}.py
# -*- coding: utf-8 -*-
import requests

YOUR_CONVERSATION_ID = ""
conversation_id = "YOUR_CONVERSATION_ID"

# 示例：GET /conversation/{conversation_id}
# 使用 GET 方法请求 /conversation/{conversation_id}
url = f"http://localhost:8000/api/v1/conversation/{conversation_id}"
headers = {"Authorization": "Bearer YOUR_TOKEN"}

try:
    response = requests.get(url, headers=headers)
    print("Status:", response.status_code)
    print("Response:", response.json())
except Exception as e:
    print("Request failed:", e)
